kart/spatial_filters.py

```diff
@@ -333,6 +335,25 @@ def split_file(cls, contents):
         return [p.strip() for p in parts]
 
 
+class MatchResult(Enum):
+    """Stores a bit more detail about whether a feature matches or doesn't match the spatial filter."""
+
+    # The given feature...
+    # Doesn't match any spatial filter because it is null / None / non-existent:
+    IS_NONE = auto()
```

Comment:
None is a Python-ey term... should we be using null for data values? Not a big deal if it's something we're doing already